setup.py

```diff
@@ -146,6 +166,13 @@ def reduce_requirements(key):
     author_email='tokui@preferred.jp',
     url='https://docs-cupy.chainer.org/',
     license='MIT License',
+    download_url="https://pypi.org/project/cupy/",
```

Comment:
It seems that other projects use this field as a link to tarballs.
https://sherif.io/2016/09/30/Python-package-with-GitHub-PyPI.html#add-tarball-download-url-to-setuppy

But it's not required and is not used even in pypa's official [example project](https://github.com/pypa/sampleproject/blob/master/setup.py). So how about removing unless if you need some reason to include this?

Reply:
OK, I will remove this line.